content/tags/control_flow_tags.md

```diff
@@ -0,0 +1,100 @@
+# Control Flow Tags
+
+Control Flow tags determine which block of code should be executed based on different conditions.
+
+1. TOC
+{:toc}
+
+## if
+
+Executes a block of code only if a certain condition is met.
+
+#### Input
+
+~~~ liquid
+  {% if product.title == 'Awesome Shoes' %}
+      These shoes are awesome!
+  {% endif %}
+~~~
+
+#### Output
+
+~~~
+  These shoes are awesome!
+~~~
+
+## elsif / else
+
+Adds more conditions within an `if` or `unless` block.
+
+#### Input
+
+~~~ liquid
+<!-- If customer.name = 'anonymous' -->
+
+  {% if customer.name == 'kevin' %}
+    Hey Kevin!
+  {% elsif customer.name == 'anonymous' %}
+    Hey Anonymous!
+  {% else %}
+    Hi Stranger!
+  {% endif %}
+~~~
+
+#### Output
+
+~~~
+  Hey Anonymous!
+~~~
+
+## case / when
+
+Creates a switch statement to compare a variable with different values. `case` initializes the switch statement, and `when` compares its values.
+
+#### Input
+
+~~~ liquid
+  {% assign handle = 'cake' %}
+  {% case handle %}
+    {% when 'cake' %}
+       This is a cake
+    {% when 'cookie' %}
+       This is a cookie
+    {% else %}
+       This is not a cake nor a cookie
+  {% endcase %}
+~~~
+
+#### Output
+
+~~~
+  This is a cake
+~~~
+
+## unless
+
+Similar to `if`, but executes a block of code only if a certain condition is **not** met.
+
+#### Input
+
+~~~ liquid
+  {% unless product.title == 'Awesome Shoes' %}
+      These shoes are not awesome.
```

Comment:
Best to change to BookingSync wording in this PR as well.

We could mention stuff about rentals, like "View from swimming pool was marvelous"
Type of amenities for string arrays etc